misc/scripts/install/docker-volume-vsphere.conf

```diff
@@ -7,4 +7,4 @@ respawn
 
 kill timeout 20
 
-exec /usr/local/bin/$UPSTART_JOB
+exec /usr/local/bin/$UPSTART_JOB -port 1019
```

Comment:
What if the config uses a different port though? Won't this hardcoding be incorrect?